devtools/td-layout-config/src/main.rs

```diff
@@ -16,6 +16,8 @@ use serde::Deserialize;
 const TD_LAYOUT_BUILD_TIME_RS_OUT: &str = "build_time.rs";
 const TD_LAYOUT_RUNTIME_RS_OUT: &str = "runtime.rs";
 
+const TD_SHIM_SEC_INFO_OFFSET: u32 = 0xFF_FFD4;
```

Comment:
Please add comment how this is calculated.

Reply:
Sure, thanks for reminder